modules/govuk_gor/manifests/init.pp

```diff
@@ -47,6 +47,20 @@
     require => Class['govuk::deploy'],
   }
 
+  cron { 'stop_gor':
+    command => 'echo "true" > /etc/govuk/env.d/FACTER_data_sync_in_progress; sudo initctl stop gor',
+    user    => $govuk_env_sync::user,
+    hour    => 23,
+    minute  => 0,
+  }
+
+  cron { 'start_gor':
+    command => 'echo "" > /etc/govuk/env.d/FACTER_data_sync_in_progress; sudo initctl start gor;',
```

Comment:
Might be clearer to extract the `/etc/govuk/env.d/FACTER_data_sync_in_progress` value in to a constant as it's used in a few places in this file?